### ISSUE TITLE
Revise cooling system activation rules

### ISSUE THREAD

**Issue by martin-mosteiro** [architecture-building-systems/CityEnergyAnalyst#1175]
Currently, the following code determines whether the system is activated or not:
```
def cooling_system_is_active(tsd, t):
    if not np.isnan(tsd['ta_cs_set'][t]) and tsd['T_ext'][t] >= tsd['ta_cs_set'][t]:
        # system has set point and other rules
        return True
    else:
        return False
```
That is, if it's the cooling season and the exterior temperature is greater than the temperature set point, the system is active and may provide cooling. The reason for using the exterior temperature is that if the exterior temperature is lower than the interior, cooling could just be provided through ventilation.

However, for buildings with radiant cooling there might not be enough ventilation to provide cooling as well.

Thus, the cooling system control should depend on system type.

**Comment by gabriel-happle** [comment]
Should I fix that @martin-mosteiro ?

**Comment by martin-mosteiro** [comment]
@gabriel-happle This was solved by PR #1189, right?
